src/vs/workbench/parts/extensions/electron-browser/extensionsActions.ts

```diff
@@ -191,6 +191,11 @@ export class InstallAction extends Action {
 				return this.notificationService.error(err);
 			}
 
+			// {{SQL CARBON EDIT}}
```

Comment:
Why do we need changes here?

Reply:
added comments

Reply:
This is already handled later by vscode;
https://github.com/Microsoft/azuredatastudio/pull/4267/files#diff-148a21c9158278aadbd155843ed47c36L64